.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dev dependencies
+      run: |
+        pip install -r requirements-dev.txt
+
+    - name: Run unit tests
+      run: |
+        python -m unittest
```

Comment:
interesting to me that it is just 'python' and not 'python3' or '$python' of some sort for the permutation.